contracts/modules/DecentSablierStreamManagementModule.sol

```diff
@@ -3,12 +3,10 @@ pragma solidity 0.8.28;
 
 import {Enum} from "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
 import {IAvatar} from "@gnosis.pm/zodiac/contracts/interfaces/IAvatar.sol";
-import {ISablierV2Lockup} from "./interfaces/sablier/full/ISablierV2Lockup.sol";
-import {Lockup} from "./interfaces/sablier/full/types/DataTypes.sol";
-
-contract DecentSablierStreamManagement {
-    string public constant NAME = "DecentSablierStreamManagement";
```

Comment:
Is there a reason removing it? Why it was here before? I kinda like having it 🤣

Reply:
Just seemed unnecessary, that's all. Why did you like having that NAME constant in this contract? AFAIK we don't use it anywhere.

Reply:
Yeah, if it wasn't used - than definitely unnecessary, just love declaratively exposing contract identification. 
But any other contract can add the very same prop 🤣